Vizzuality/heco-invest
# Use the `GeometryInput` component on the project creation form

This PR completes the `<GeometryInput />` component with a map the user can draw on and adds it to the project creation form.

The search field that is also part of the input will be dealt with in a separate user story ([LET-376](https://vizzuality.atlassian.net/browse/LET-376)).

## Testing instructions

Try to create a new project. Make sure to specify a ticket size otherwise the API will crash (see [LET-377](https://vizzuality.atlassian.net/browse/LET-377)).

## Tracking

[LET-348](https://vizzuality.atlassian.net/browse/LET-348).
